editor/animation_track_editor.cpp

```diff
@@ -6303,6 +6313,18 @@ void AnimationTrackEditor::_show_imported_anim_warning() {
 			TTR("Warning: Editing imported animation"));
 }
 
+void AnimationTrackEditor::_show_dummy_player_warning() {
+	EditorNode::get_singleton()->show_warning(
+			TTR("Some AnimationPlayerEditor's options are disabled since this is the dummy AnimationPlayer for preview.\n\nThe dummy player is forced active, non-deterministic and doesn't have the root motion track. Furthermore, the original node is inactive temporary."),
+			TTR("Warning: Preview dummy AnimationPlayer"));
+}
+
+void AnimationTrackEditor::_show_inactive_player_warning() {
+	EditorNode::get_singleton()->show_warning(
+			TTR("AnimationPlayer is inactive. The playback result is not applied to the scene."),
+			TTR("Warning: AnimationPlayer is inactive"));
```

Comment:
```suggestion
			TTR("Warning"));
```
I think Warning is sufficient, as the same thing is said in the message.
Same above.